timmy/tools.py

```diff
@@ -46,6 +46,32 @@
 '''
 
 
+def auth_token(conf):
+    req_data = json.dumps(
+        {'auth':
+            {'scope':
+                {'project':
+                    {'domain': {'id': 'default'},
+                     'name': conf['tenant']}},
+                     'identity':
+             {'password':
+                 {'user':
+                     {'domain': {'id': 'default'},
+                      'password': conf['fuel_pass'],
+                      'name': conf['fuel_user']}},
+                      'methods': ['password']}
+             }
+         }
+    )
+
+    req = urllib2.Request("http://" + conf['fuel_ip'] + ":"
```

Comment:
I recommend rewriting to use sting expansion:
`'http://%s:%s....' % (x,y,z...)`